src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -895,6 +936,28 @@ private static bool IsBlobFeedUrl(string location)
             return locationUri.Host.EndsWith("blob.core.windows.net") && location.EndsWith("/index.json");
         }
 
+        /// <summary>
+        ///     Determine whether this location is an Azure DevOps Feed URL.
+        /// </summary>
+        /// <param name="location">Location</param>
+        /// <returns>True if the location is an AzDO Feed URL, false otherwise.</returns>
+        /// <remarks>
+        ///     AzDO feed uris look like: 
+        ///         - https://pkgs.dev.azure.com/dnceng/public/_packaging/public-feed-name/nuget/v3/index.json
+        ///         - https://pkgs.dev.azure.com/dnceng/_packaging/internal-feed-name/nuget/v3/index.json
+        /// </remarks>
+        private static bool IsAzureDevOpsFeedUrl(string location, out string feedAccount, out string feedVisibility, out string feedName)
```

Comment:
This method looks like it's a parse + validate. Would it make sense to rename it to ParseAzureDevOpsFeedUrl that returns a tuple of the out variables you are passing here?

Reply:
Agreed.

Reply:
Actually, I think it's fine as is...it matches pretty well the rest of the if/else pattern, returning a bool.